agent/agents/mongodb/internal/profiler/collector/extended.go

```diff
@@ -0,0 +1,23 @@
+// Copyright (C) 2023 Percona LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package collector
+
+import "github.com/percona/percona-toolkit/src/go/mongolib/proto"
+
+// ExtendedSystemProfile is extended SystemProfile from Percona Toolkit.
+type ExtendedSystemProfile struct {
+	proto.SystemProfile `bson:",inline"`
```

Comment:
Is the comma not a typo here?

Reply:
I believe that without that proto.SystemProfile properties wont be on same (TOP) level in struct.